packages/health/lib/src/health_data_point.dart

```diff
@@ -12,17 +12,10 @@ class HealthDataPoint {
   String _deviceId;
   String _sourceId;
   String _sourceName;
+  String _uuid;
```

Comment:
Is there a specific reason to add this new field?

Reply:
@hoffmatteo added only for simplify cuz plugin is not returning an uuid in general this means that we need to use another plugins to get uuid of user. In general this is more a draft which you can change if needed